localized/games/wordwindow/wordwindow_level_bn.tsv

```diff
@@ -1,241 +1,241 @@
 #languageTag	#level	#worksheet	#problemNo	#sequenceType	#voiceSource	#problemSentence	#exampleNumber1	#exampleNumber2	#exampleNumber3	#exampleNumber4	#exampleWord1	#exampleWord2	#exampleWord3	#exampleWord4	#rightFormula	#rightAnswer	#objectValueA	#objectValueB	#objectValueC	#objectValueD	#equationValueA	#equationValueB	#equationSign	#equationUnknown	#objectSourceA	#objectSourceB	#objectSourceC	#objectSourceD
-en-US	1	1	1	1	জ্যাকের মুরগি গতকাল 6 টি এগ্স  এবং আজ 3 টি এগ্স  দিয়েছে. জ্যাকের মুরগি  কত এগ্স দিয়েছে?	জ্যাকের মুরগি গতকাল 6 টি এগ্স  এবং আজ 3 টি এগ্স  দিয়েছে. জ্যাকের মুরগি  কত এগ্স দিয়েছে?	6	7	8	9	eggs	eggs	eggs	eggs	6 + 3 = <9>	4	6	3	0	0					wordwindow-stone-yellow.png	wordwindow-stone-red.png		
-en-US	1	1	2	1	2 টি বার্ডস ফেন্স এ   বসে ছিল. আরো 4 বার্ডস  এসে বসলো. ফেন্স  এ কত birds আছে ?	2 টি বার্ডস ফেন্স এ   বসে ছিল. আরো 4 বার্ডস  এসে বসলো. ফেন্স  এ কত birds আছে ?	6	7	8	9	birds	birds	birds	birds	2 + 4 = <6>	1	2	4	0	0					wordwindow-stone-red.png	wordwindow-stone-blue.png		
-en-US	1	1	3	1	স্কুলে গণিত করার সময় শেলবি স্টার স্টিকারস পায়ে. গতকাল, শেলবি 4 স্টার্স  পেলো. আজ আর 3 পেয়েছে. শেলবি কত স্টার্স পেলো? 	স্কুলে গণিত করার সময় শেলবি স্টার স্টিকারস পায়ে. গতকাল, শেলবি 4 স্টার্স  পেলো. আজ আর 3 পেয়েছে. শেলবি কত স্টার্স পেলো? 	6	7	8	9	stars	stars	stars	stars	4 + 3 = <7>	2	4	3	0	0					wordwindow-stone-blue.png	wordwindow-stone-green.png		
+en-US	1	1	1	1	জ্যাকের মুরগি গতকাল 6 টি ডিম  এবং আজ 3 টি ডিম  দিয়েছে | জ্যাকের মুরগি  কত ডিম দিয়েছে?	জ্যাকের মুরগি গতকাল 6 টি ডিম  এবং আজ 3 টি ডিম  দিয়েছে | জ্যাকের মুরগি  কত ডিম দিয়েছে?	6	7	8	9	eggs	eggs	eggs	eggs	6 + 3 = <9>	4	6	3	0	0					wordwindow-stone-yellow.png	wordwindow-stone-red.png		
```

Comment:
remove extra spaces between ` |`. Also this is pipe and not the purnaviram symbol